.github/workflows/build.yml

```diff
@@ -16,11 +16,11 @@ jobs:
     - name: Setup .NET Core
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 5.0.100-preview.7.20366.6
+        dotnet-version: 5.0.100-preview.8.20417.9
     - name: Test
-      run: dotnet test --verbosity normal
+      run: ./build.cmd test
     - name: Pack
-      run: dotnet pack --configuration Release --version-suffix build$GITHUB_RUN_NUMBER.0 -p:ContinuousIntegrationBuild=true
+      run: ./build.cmd pack
```

Comment:
You aren't marking CI builds accordingly. Please pass in `build$GITHUB_RUN_NUMBER.0` as the version suffix.

Reply:
You're also not setting the ContinuousIntegrationBuild property